docs/jargon/mesh.md

```diff
@@ -80,6 +80,69 @@ An [**Activity**](#activity) that is invoked directly in the same process by Wor
 
 - Although a Local Activity consumes less resources than a normal [**Activity**](#activity), it is subject to shorter durations and a lack of rate limiting.
 
+## Retry Policy
+
+A collection of attributes that instructs Temporal Server how to retry a failure of an Activity Task Execution or a [**Workflow Execution**](#workflow-execution).
+Any custom Retry Policy must be provided when an [**Activity Task**](#activity-task) or Workflow starts.
+
+Retrying an [**Activity Task**](#activity-task) produces a new Activity Task Execution that uses the same parameters [VERIFY] and Task Queue as the previous attempt.
+
+An Activity Task Execution has a default Retry Policy and thus retries by default.
+
+A [**Workflow Execution**](#workflow-execution) does not have a default Retry Policy and thus does not retry by default.
+The intention is that a Workflow is written to never fail due to intermittent issues; an Activity is designed to handle such issues.
+(Be aware that, by default, a Workflow Task Execution does retry indefinitely.)
+
+A [**Workflow Execution**](#workflow-execution) should be retried only in specific scenarios, such as the following:
+
+- A Cron Workflow or some other stateless always-running Workflow Execution that can benefit from retries.
+- A file-processing or media-encoding Workflow Execution that downloads files to a host.
+
+A Retry Policy does not apply to a Workflow Task Execution, which is governed by the Schedule-To-Start timeout.
+
+The time between retries is the _retry interval_. A retry interval is the smaller of two values:
```

Comment:
```suggestion
The time between retries is the _retry interval_.
A retry interval is the smaller of two values:
```

Reply:
Shall we include the simple diagram here?

Reply:
Makes sense to me. @dtornow?